scripts/ci/travis/install

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-# This script will build the project.
```

Comment:
can you add a one line comment to each of the scripts to indicate what it is and why it should be run?